source/server/admin/prometheus_stats_formatter.cc

```diff
@@ -0,0 +1,84 @@
+#include "source/server/admin/prometheus_stats_formatter.h"
+
+#include "source/common/common/empty_string.h"
+#include "source/common/common/macros.h"
+#include "source/common/common/regex.h"
+#include "source/common/stats/histogram_impl.h"
+
+#include "absl/strings/str_cat.h"
+#include "absl/strings/str_replace.h"
+
+namespace Envoy {
+namespace Server {
+
+namespace {
+
```

Comment:
I assume this code is unchanged from what was in prometheus_stats.cc.

Actually would it make more sense to merge this with the prometheus render code below?

Reply:
Yep, this is pre-existing code: sure, I will move it to the `PrometheusStatsRender` class.